wandb/apis/public.py

```diff
@@ -2531,6 +2531,46 @@ def aliases(self, aliases):
                 )
         self._aliases = aliases
 
+    @staticmethod
+    def expected_type(client, name, entity_name, project_name):
```

Comment:
Do we need this? I thought this would be equivalent to:

```
public_api.artifact(name).type
```

Reply:
Oh, I guess the difference is in how it handles the case where the artifact doesn't exist, which makes sense.

Reply:
Yeah, which could be handled in a try/except.